### PR TITLE
Fix gm hermes keys command when hdpath is set

### PR DESCRIPTION
**Pull request by greg-szabo**
Closes: #975

## Description

* Fixes `gm hermes keys` command when the `wallet_hdpath` contains a non-default coin type.
* Adds a changelog to the gm tool for easier tracking.
* bumps version to 0.0.2

______

For contributor use:

- [X] Updated the __Unreleased__ section of [CHANGELOG.md](https://github.com/informalsystems/ibc-rs/blob/master/CHANGELOG.md) with the issue.
- [ ] If applicable: Unit tests written, added test to CI.
- [X] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation (`docs/`) and code comments.
- [X] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### BUG FIXES
+- [#975] gm tool: Import hermes keys properly even if wallet HD derivation path is set.
```
Comment (romac) [inline CHANGELOG.md:4]:
```suggestion
- [gaia-manager]
  - [#975] Import hermes keys properly even if wallet HD derivation path is set.
```

File: scripts/gm/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Introduced [CHANGELOG](https://github.com/informalsystems/ibc-rs/blob/master/scripts/gm/CHANGELOG.md) file.
+
+[#975]: https://github.com/informalsystems/ibc-rs/issues/975
+
+## v0.0.1
+
+FEATURES
+- [#902] Initial release
+- [#README](https://github.com/informalsystems/ibc-rs/blob/1d0935f3b4616f5562f68a2bba0b4b5a6648aecf/scripts/gm/README.md)
+Feature list
```
Comment (romac) [inline scripts/gm/CHANGELOG.md:18]:
```suggestion
### BUGFIXES
- Import hermes keys properly even if hdpath is set ([#975])

### FEATURES
- Introduced [CHANGELOG](https://github.com/informalsystems/ibc-rs/blob/master/scripts/gm/CHANGELOG.md) file.

[#975]: https://github.com/informalsystems/ibc-rs/issues/975

## v0.0.1

FEATURES
- [#902] Initial release
- [#README](https://github.com/informalsystems/ibc-rs/blob/1d0935f3b4616f5562f68a2bba0b4b5a6648aecf/scripts/gm/README.md)
```

**Comment by romac** [comment]
Do we want to the `keys add` command to take an option to specify the full hd path rather than just the coin type? Or is the latter fine for now?

**Comment by greg-szabo** [comment]
Fine for now. I'll open a hermes ticket if we realize we need the full HD path in Hermes.

**Review by ancazamfir — APPROVED** [review]

**Review by adizere — APPROVED** [review]

**Pull request merged**